doc/trex_astf.asciidoc

```diff
@@ -3915,7 +3915,7 @@ link:cp_astf_docs/index.html[python index]
 |   tcp.keepidle    |2-65533|5|-|+| value in second for TCP keepidle
 |   tcp.keepintvl   |2-65533|7|-|+| value in second for TCP keepalive interval
 |   tcp.blackhole   |0,1,2|0|-|+| 0 - return RST packet in case of error. 1- return of RST only in SYN. 2- don't return any RST packet, make a blackhole.
-|   tcp.delay_ack_msec | 20-50000|100|-|+| delay ack timeout in msec.  Reducing this value will reduce the performance but will reduce the active flows
+|   tcp.delay_ack_msec | 1-50000|100|-|+| delay ack timeout in msec.  Reducing this value will reduce the performance but will reduce the active flows
```

Comment:
I wonder if we should keep the interval between 20 and 50000 and then write later in the description that it is possible to have values lower than 20 and higher than 0 but it is not recomended.